dipper/sources/MPD.py

```diff
@@ -453,6 +454,12 @@ def _add_g2p_assoc(self, g, strain_id, sex, assay_id, phenotypes, comment):
                 assoc.add_association_to_graph()
                 assoc_id = assoc.get_association_id()
                 model.addComment(assoc_id, comment)
+                if sex == 'm':
+                    model._addSexSpecificity(assoc_id,
+                                             self.global_terms['male'])
+                elif sex == 'f':
+                    model._addSexSpecificity(assoc_id,
+                                             self.global_terms['female'])
```

Comment:
this could be set up with local  having `'m': 'male'`   &  `'f' : 'female' ` 
then the stanza becomes  the line:  
 
     model._addSexSpecificity(assoc_id, self.resolve(sex))